0-generate-pkg-list

```diff
@@ -3,6 +3,22 @@
 # This script promotes osg-version to testing and generates a preliminary list of packages for the release manager
 
 . release-common.sh
+
+# make sure osg-koji is available before starting
+check_for_osg_koji
+
+release_tools="osg-pkgs-behind-tag"
+
+for i in $release_tools;
```

Comment:
Nitpick: you only need the semicolon if the `do` is on the same line.

Reply:
Fixed